docs/source/contribution/technical_steering_committee.md

```diff
@@ -52,16 +52,15 @@ We look for commitment markers who can do the following:
 
 <!-- DO NOT EDIT THIS AND MERGE A PR WITHOUT A VOTE TO SIGN OFF ANY CHANGES -->
 
-Kedro was originally designed by [Aris Valtazanos](https://github.com/arisvqb) and [Nikolaos Tsaousis](https://github.com/tsanikgr) at QuantumBlack to solve challenges they faced in their project work. Their work was later turned into an internal product by [Peteris Erins](https://github.com/Pet3ris), [Ivan Danov](https://github.com/idanov), [Nikolaos Kaltsas](https://github.com/nikos-kal), [Meisam Emamjome](https://github.com/misamae) and [Nikolaos Tsaousis](https://github.com/tsanikgr).
+Kedro was originally designed by [Aris Valtazanos](https://github.com/arisvqb) and [Nikolaos Tsaousis](https://github.com/tsanikgr) at [QuantumBlack](https://www.mckinsey.com/capabilities/quantumblack) to solve challenges they faced in their project work. Their work was later turned into an internal product by [Peteris Erins](https://github.com/Pet3ris), [Ivan Danov](https://github.com/idanov), [Nikolaos Kaltsas](https://github.com/nikos-kal), [Meisam Emamjome](https://github.com/misamae) and [Nikolaos Tsaousis](https://github.com/tsanikgr).
 
 
-Currently, the core Kedro team consists of:
+The TSC comprises the following team members from [QuantumBlack Labs](https://www.mckinsey.com/capabilities/quantumblack/labs):
```

Comment:
I'm not a fan of the QB vs other split, because I feel like it makes the organization dominance more explicit. If the goal is to be presented as an open source project more than a QB project, I think it makes sense to look more like the Delta example @astrojuanlu shared, and list affiliation for each person. That way, I interpret it as:

* Ahdra is a maintainer, who happens to work at QB
* Deepyaman is a maintainer, who happens to work at Claypot
* Mackay is a maintainer, who happens to work at QB
* Marcin is a maintainer, who happens to work at GetInData

Rather than:

* QB maintains Kedro; team consists of Ahdra, Mackay, etc.
* Deepyaman (from Claypot) and Marcin (from GetInData) also contribute

Reply:
I take your point but also it gets very repetitive very swiftly if we add in all the QB people. I don't personally see that the "other" folks look like afterthoughts, but if you do, you could move them ahead of QB or add a sentence about how much we value additional non-QB members on the TSC and if you want to join, please read the above, blah blah, to volunteer yourself for a member.

If this is going to cause more problems than solutions, let's just revert to what we have or someone else can drive this 🤷

Reply:
> it makes the organization dominance more explicit.

That's just how it is though, why minimize it? Undoubtedly at this stage it'll be like this until we grow out of it.

I like your changes, Jo, and also agree that it gets repetitive too quickly otherwise.

Reply:
I'm with Deepyaman on this one. Could we have a table instead, sorted by person name? So it's not a dense list of links and every organization is given the same importance.

Of course QB has a "special place" because we sort of control lots of the assets and everybody from the team is part of the TSC, but I think this page should go in the direction of trying to minimize that difference, not exacerbate it.

Reply:
Thanks all for your feedback. I've decided to set this aside for now as it's low priority compared to other tasks this sprint. I've reverted all but the addition of a @MehdiNV and a link to QB. We can pick this up again when it's practical.